azure-py-webserver/__main__.py

```diff
@@ -0,0 +1,78 @@
+import pulumi
+from pulumi import Output
+from pulumi_azure import core, compute, network
+
+config = pulumi.Config("azure-web")
+username = config.require("username")
+password = config.require("password")
+
+resource_group = core.ResourceGroup("server", location="West US")
+net = network.VirtualNetwork("server-network",
+                             resource_group_name=resource_group.name,
+                             location=resource_group.location,
+                             address_spaces=["10.0.0.0/16"],
+                             subnets=[{
+                                 "name": "default",
+                                 "address_prefix": "10.0.1.0/24",
+                             }])
```

Comment:
Is this indentation style our standard?  This looks unusual to me - but maybe I'm just not up to speed on the latest Python coding style?

Reply:
This is PyCharm's default. It's not pep8; I can run this through a code formatter and see how it looks.